File: Tip_Calculator.py
def tip_calculator():
    bill = float(input("Enter the total bill amount: "))
    tip_percentage = float(input("Enter the tip percentage: "))
    num_people = int(input("Enter the number of people: "))

    tip_amount = bill * (tip_percentage / 100)
    total_bill = bill + tip_amount
    bill_per_person = total_bill / num_people

    print("Total bill: ", total_bill)
    print("Bill per person: ", bill_per_person)

tip_calculator()

File: Tip_Calculator.java
```java
import java.util.Scanner;

public class TipCalculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the bill amount: ");
        double bill = scanner.nextDouble();
        System.out.println("Enter the tip percentage: ");
        double tipPercentage = scanner.nextDouble();
        double tip = bill * (tipPercentage / 100);
        double total = bill + tip;
        System.out.println("The tip amount is: " + tip);
        System.out.println("The total bill amount is: " + total);
    }
}
```